docs/src/main/paradox/server/reflection.md

```diff
@@ -27,17 +27,31 @@ can be used to query this information
 
 ## Providing
 
-When providing gRPC Server Reflection, you will be serving multiple services
-(at least your own service and the reflection service) as described
-@ref[here](walkthrough.md) { #Serving_multiple_services }. The reflection service
-can be generated via `ServerReflection`:
+The generated service handler includes a convenience method to create an Akka HTTP 
+handler with your service with a `ServerReflection` handler fallback: 
```

Comment:
Good point, it doesn't make sense to present this feature in terms of how it's implemented.

At the same time, since this is a convenience method (_ehem_ a `Utils` method _/ehem_), it's not so bad to expose how it is implemented. 

I'll continue thinking of other options anyway. :-)